tests/test_environment_toctree.py

```diff
@@ -16,23 +16,13 @@
 from sphinx.builders.html import StandaloneHTMLBuilder
```

Comment:
Review is **NOT** needed.
- Converting gen_with_app testing into individual (but sahreing build result) tests